msgid ""
msgstr ""
"Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-07-13 20:53+0200\n"
"PO-Revision-Date: 2015-01-18 18:30-0500\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Portuguese, Brazilian\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: pyload\n"
"X-Crowdin-File: /pyLoad/core.pot\n"

#: pyLoadCore.py:181
msgid "Received Quit signal"
msgstr "sinal de Sair recebido"

#: pyLoadCore.py:302
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad já está rodando com pid %s"

#: pyLoadCore.py:316
#, python-format
msgid "Failed changing group: %s"
msgstr "Falha ao trocar grupo: %s"

#: pyLoadCore.py:326
#, python-format
msgid "Failed changing user: %s"
msgstr "Falha ao trocar de usuario: %s"

#: pyLoadCore.py:328
msgid "folder for logs"
msgstr "pasta para logs"

#: pyLoadCore.py:339
msgid "Starting"
msgstr "Iniciando"

#: pyLoadCore.py:340
#, python-format
msgid "Using home directory: %s"
msgstr "Usando diretório: %s"

#: pyLoadCore.py:349
msgid "pycrypto to decode container files"
msgstr "pycrypto para decodificar arquivos do contêiner"

#: pyLoadCore.py:352
msgid "folder for temporary files"
msgstr "pasta para arquivos temporários"

#: pyLoadCore.py:357
msgid "folder for downloads"
msgstr "pasta para downloads"

#: pyLoadCore.py:360
msgid "OpenSSL for secure connection"
msgstr "OpenSSL para conexão segura"

#: pyLoadCore.py:364
msgid "Moving old user config to DB"
msgstr "Movendo a configuração de utilizador antiga para BD"

#: pyLoadCore.py:367
msgid "Please check your logindata with ./pyLoadCore.py -u"
msgstr "Por favor verificar dados de login com  ./pyLoadCore.py -u"

#: pyLoadCore.py:370
msgid "All links removed"
msgstr "Todos os links foram removidos"

#: pyLoadCore.py:401
#, python-format
msgid "Downloadtime: %s"
msgstr "Tempo de download: %s"

#: pyLoadCore.py:411
#, python-format
msgid "Free space: %s"
msgstr "Espaço livre: %s"

#: pyLoadCore.py:431
msgid "Activating Accounts..."
msgstr "Ativando contas..."

#: pyLoadCore.py:437
msgid "Activating Plugins..."
msgstr "Ativar Plugins..."

#: pyLoadCore.py:440
msgid "pyLoad is up and running"
msgstr "pyLoad está rodando"

#: pyLoadCore.py:459
msgid "restarting pyLoad"
msgstr "reiniciando pyLoad"

#: pyLoadCore.py:463
msgid "pyLoad quits"
msgstr "pyLoad fechou"

#: pyLoadCore.py:520
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: pyLoadCore.py:556
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "não foi possível encontrar %(desc)s: %(name)s"

#: pyLoadCore.py:558
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "não foi possível criar %(desc)s: %(name)s"

#: pyLoadCore.py:579
msgid "shutting down..."
msgstr "desligando..."

#: pyLoadCore.py:596
msgid "error while shutting down"
msgstr "erro durante desligamento"

#: pyLoadCore.py:660
msgid "killed pyLoad from Terminal"
msgstr "pyLoad fechado pelo terminal"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "A base de dados dos ficheiros foi eliminada devido a incompatibilidade de versões."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "Não foi possível converter a base de dados dos ficheiros."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "A base de dados foi convertida da v2 para a v3."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "A base de dados foi convertida da v3 para a v4."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Convertendo BD Django antigo"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "Terminado"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr "Offline"

#: module/database/FileDatabase.py:45
msgid "online"
msgstr "Online"

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "em fila"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "ignorada"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "esperando"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "Temp. off-line"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "iniciando"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "falhou"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "abortada"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "descriptografando"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "personalizado"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "Transferindo"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "processando"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "desconhecido"

#: module/database/FileDatabase.py:531 module/plugins/hooks/IRCInterface.py:74
#: module/plugins/hooks/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Pacote finalizado: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Usando o SSL ThriftBackend"

#: module/remote/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Erro de back-end remoto: %s"

#: module/remote/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Iniciando %(name)s: %(addr)s:%(port)s"

#: module/remote/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Falha ao carregar back-end %(name)s | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "à espera%s"

#: module/web/ServerThread.py:36
msgid "SSL certificates not found."
msgstr "Certificado SSL não encontrado."

#: module/web/ServerThread.py:40
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Desculpa, retirámos o suporte para iniciar %s diretamente a partir do pyLoad"

#: module/web/ServerThread.py:41
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "O servidor threaded pode ser utilizado, oferecendo bom desempenho e ssl,"

#: module/web/ServerThread.py:42
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "Claro, você ainda pode usar o %s existente com o servidor fastcgi do pyLoad"

#: module/web/ServerThread.py:43
msgid "sample configs are located in the module/web/servers directory"
msgstr ""

#: module/web/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Não é possível usar o %(server)s, python-flup não está instalado!"

#: module/web/ServerThread.py:58
#, python-format
msgid "Error importing lightweight server: %s"
msgstr ""

#: module/web/ServerThread.py:59
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Você precisa baixar e compilar bjoern, https://github.com/jonashaag/bjoern"

#: module/web/ServerThread.py:60
msgid "Copy the boern.so to module/lib folder or use setup.py install"
msgstr "Copie o boern.so para a pasta lib/módulo ou use o instalador setup.py"

#: module/web/ServerThread.py:61
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "É claro que você precisa estar familiarizado com linux e saber como compilar software"

#: module/web/ServerThread.py:64
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Servidor definido como threaded, devido a problemas de desempenho conhecidos no windows."

#: module/web/ServerThread.py:80 module/web/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Este servidor não oferece SSL, por favor, considere usar HTTP{?} em vez disso"

#: module/web/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Iniciando o servidor web embutido: %(host)s:%(port)d"

#: module/web/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr ""

#: module/web/pyload_app.py:127
msgid "You dont have permission to access this page."
msgstr ""

#: module/web/pyload_app.py:195
msgid "Download directory not found."
msgstr ""

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "unlimited"
msgstr "ilimitado"

#: module/web/pyload_app.py:264 module/web/pyload_app.py:271
msgid "not available"
msgstr "Indisponível"

#: module/web/pyload_app.py:509
msgid "Run pyLoadCore.py -s to access the setup."
msgstr ""

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "pedaços do Download falharam, retornar para conexão simples |%s"

#: module/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Inicio do Download: %s"

#: module/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Termino do Download: %s"

#: module/PluginThread.py:194 module/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "Plugin %s está sem uma função."

#: module/PluginThread.py:202 module/PluginThread.py:265
#: module/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Download abortado: %s"

#: module/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Download reiniciado: %(name)s | %(msg)s"

#: module/PluginThread.py:231 module/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "Download está offline: %s"

#: module/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "Download esta temporariamente offline: %s"

#: module/PluginThread.py:237 module/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Download falhou: %(name)s | %(msg)s"

#: module/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Impossível conectar com o host ou a conexão receptou, tentar novamente em 1 minuto."

#: module/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Download pulado: %(name)s pois %(plugin)s"

#: module/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr ""

#: module/PluginThread.py:376 module/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr ""

#: module/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr ""

#: module/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Busca de informações para %(name)s falhou|%(err)s"

#: module/HookManager.py:90 module/plugins/Hook.py:103
#, python-format
msgid "Error executing hooks: %s"
msgstr ""

#: module/HookManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Falha de ativação %(name)s"

#: module/HookManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Ativado plugins: %s"

#: module/HookManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Desactivar plugins: %s"

#: module/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Reconectar Falhou: %s"

#: module/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Script de reconexão não encontrado!"

#: module/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Iniciando reconexão"

#: module/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Falha ao executar script de reconexão!"

#: module/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Reconectado, novo IP: %s"

#: module/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Espaço insuficiente no dispositivo"

#: module/plugins/Account.py:85 module/plugins/Account.py:90
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Não pode ligar com a conta %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Senha incorreta"

#: module/plugins/Account.py:243
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr ""

#: module/plugins/Account.py:269
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "A conta %s não tem tráfego suficiente, checando novamente em 30 min"

#: module/plugins/Account.py:276
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "A conta %s expirou, checando novamente em 1h"

#: module/plugins/crypter/SerienjunkiesOrg.py:126
msgid "Downloadlimit reached"
msgstr "Limite de Download alcançado"

#: module/plugins/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr ""

#: module/plugins/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Erro ao importar %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:132
msgid "No Hoster loaded"
msgstr "Nenhum Hoster carregado"

#: module/plugins/accounts/BitshareCom.py:38
msgid "Activate direct Download in your Bitshare Account"
msgstr "Ativar Download direto em sua conta Bitshare"

#: module/plugins/container/LinkList.py:64
msgid "LinkList could not be cleared."
msgstr ""

#: module/plugins/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Configurações da conta excluídas, devido ao novo formato de configuração."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorização necessaria (nome de usuario: senha)"

#: module/plugins/hoster/SimplydebridCom.py:23
#: module/plugins/hoster/RealdebridCom.py:40
#: module/plugins/hoster/FreeWayMe.py:39 module/plugins/hoster/ZeveraCom.py:21
#: module/plugins/hoster/UnrestrictLi.py:52
#: module/plugins/hoster/Vipleech4uCom.py:30
#: module/plugins/hoster/Premium4Me.py:27 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/SimplyPremiumCom.py:52
#: module/plugins/hoster/MegaDebridEu.py:46
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/LinksnappyCom.py:29
#: module/plugins/hoster/DebridItaliaCom.py:39
#: module/plugins/hoster/RPNetBiz.py:28
#: module/plugins/hoster/MultiDebridCom.py:40
#: module/plugins/hoster/ReloadCc.py:26 module/plugins/hoster/OverLoadMe.py:38
#: module/plugins/hoster/RehostTo.py:25
#: module/plugins/hoster/PremiumizeMe.py:24
#: module/plugins/hooks/RPNetBiz.py:45
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Por favor entre sua %s conta ou desative este plugin"

#: module/plugins/hoster/FilesMailRu.py:98
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Havia código HTML no arquivo baixado (%s)...erro de redirecionamento? O Download vai ser reiniciado."

#: module/plugins/hoster/NetloadIn.py:145
#: module/plugins/hoster/NetloadIn.py:169
msgid "File temporarily not available"
msgstr "Arquivo temporariamente indisponível"

#: module/plugins/hoster/NetloadIn.py:182
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: esperar entre os Downloads %d s."

#: module/plugins/hoster/NetloadIn.py:213
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: esperando por capacha %d s."

#: module/plugins/hoster/NetloadIn.py:251
msgid "Downloaded File was empty"
msgstr "Arquivo baixado estava vazio"

#: module/plugins/hoster/MegaDebridEu.py:49
#, python-format
msgid "Impossible to connect to %s"
msgstr "Impossível conectar-se a %s"

#: module/plugins/hoster/MegaDebridEu.py:89
#, python-format
msgid "Impossible to debrid %s"
msgstr ""

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Chave de API inválida"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Tráfego restante insuficiente"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Tráfego excedido"

#: module/plugins/hoster/RapidshareCom.py:99
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Compartilhamento de tráfego (Download direto)"

#: module/plugins/hoster/RapidshareCom.py:126
#: module/plugins/hoster/RapidshareCom.py:193
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Ja está baixando deste endereço de ip, espere 60 segundos"

#: module/plugins/hoster/RapidshareCom.py:130
msgid "Invalid Auth Code, download will be restarted"
msgstr "Código de autenticação invalido, Download vai ser reiniciado"

#: module/plugins/hoster/RapidshareCom.py:198
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Sem slots vazios"

#: module/plugins/hoster/RapidshareCom.py:201
msgid "You need a premium account for this file"
msgstr "Você precisa de uma conta premium para este arquivo"

#: module/plugins/hoster/RapidshareCom.py:203
msgid "Filename reported invalid"
msgstr "Nome do arquivo relatado como invalido"

#: module/plugins/hoster/FileserveCom.py:100
msgid "Parallel download error, now waiting 60s."
msgstr "Erro de Download paralelo, esperar 60 segundos."

#: module/plugins/hoster/FileserveCom.py:216
msgid "Not logged in."
msgstr "Não está logado."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Descriptografia falhou"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Nenhum arquivo chave fornecido na URL"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Codigo do erro:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "O arquivo não existe."

#: module/plugins/hooks/UpdateManager.py:97
msgid "Not able to connect server to get updates"
msgstr "Não é capaz de se conectar a servidor para obter as atualizações"

#: module/plugins/hooks/UpdateManager.py:111
msgid "No pyLoad version available"
msgstr "Nenhuma versão do pyLoad disponível"

#: module/plugins/hooks/UpdateManager.py:118
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "* * * Nova versão %s do pyLoad disponível * * *"

#: module/plugins/hooks/UpdateManager.py:119
msgid "***  Get it here: https://github.com/pyload/pyload/releases  ***"
msgstr "***  Consiga o aqui: https://github.com/pyload/pyload/releases  ***"

#: module/plugins/hooks/UpdateManager.py:170
#, python-format
msgid "New version of [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"
msgstr "Nova versão do [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"

#: module/plugins/hooks/UpdateManager.py:180
#: module/plugins/hooks/UpdateManager.py:185
#, python-format
msgid "Error when updating plugin %s"
msgstr "Erro ao atualizar plugin %s"

#: module/plugins/hooks/UpdateManager.py:185
msgid "Version mismatch"
msgstr "Incompatibilidade de versão"

#: module/plugins/hooks/UpdateManager.py:196
#, python-format
msgid "Removed blacklisted plugin: [%(type)s] %(name)s"
msgstr "Plugin removido da lista negra: [%(type)s] %(name)s"

#: module/plugins/hooks/UpdateManager.py:204
msgid "Plugins updated and reloaded"
msgstr "Plugins atualizados e recarregados"

#: module/plugins/hooks/UpdateManager.py:206
msgid "*** Plugins have been updated, pyLoad will be restarted now ***"
msgstr "*** Plugins foram atualizados, pyLoad será reiniciado agora ***"

#: module/plugins/hooks/UpdateManager.py:211
msgid "No plugin updates available"
msgstr "Nenhuma atualização de plugin disponível"

#: module/plugins/hooks/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Download concluído: %(name)s @ %(plugin)s "

#: module/plugins/hooks/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nova solicitação de Captcha: %s"

#: module/plugins/hooks/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Responder com 'c %s o texto no captcha'"

#: module/plugins/hooks/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Por favor adicione sua conta premium.to antes de reiniciar o pyLoad"

#: module/plugins/hooks/HotFolder.py:81
#, python-format
msgid "Added %s from HotFolder"
msgstr "Adicionado %s de HotFolder"

#: module/plugins/hooks/ExtractArchive.py:94
#, python-format
msgid "No %s installed"
msgstr "Sem %s instalado"

#: module/plugins/hooks/ExtractArchive.py:96
#: module/plugins/hooks/ExtractArchive.py:101
#, python-format
msgid "Could not activate %s"
msgstr "Não foi possível activar o %s"

#: module/plugins/hooks/ExtractArchive.py:106
msgid "Activated"
msgstr "Ativado"

#: module/plugins/hooks/ExtractArchive.py:108
msgid "No Extract plugins activated"
msgstr "Não ha plugins de extração ativos"

#: module/plugins/hooks/ExtractArchive.py:120
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Pacote %s enfileirados para extração posterior"

#: module/plugins/hooks/ExtractArchive.py:143
#, python-format
msgid "Check package %s"
msgstr "Verificar pacote %s"

#: module/plugins/hooks/ExtractArchive.py:184
#, python-format
msgid "Extract to %s"
msgstr "Extrair para %s"

#: module/plugins/hooks/ExtractArchive.py:199
msgid "No files found to extract"
msgstr "Arquivos não encontrados para extrair"

#: module/plugins/hooks/ExtractArchive.py:206
msgid "extracting"
msgstr "extraindo"

#: module/plugins/hooks/ExtractArchive.py:217
msgid "Password protected"
msgstr "Protegido por senha"

#: module/plugins/hooks/ExtractArchive.py:238
msgid "Wrong password"
msgstr "Senha incorreta"

#: module/plugins/hooks/ExtractArchive.py:246
#, python-format
msgid "Deleting %s files"
msgstr "Excluindo %s arquivos"

#: module/plugins/hooks/ExtractArchive.py:253
msgid "Extracting finished"
msgstr "Extração terminada"

#: module/plugins/hooks/ExtractArchive.py:259
msgid "Archive Error"
msgstr "Erro de arquivo"

#: module/plugins/hooks/ExtractArchive.py:261
msgid "CRC Mismatch"
msgstr "Incompatibilidade de CRC"

#: module/plugins/hooks/ExtractArchive.py:265
msgid "Unknown Error"
msgstr "Erro desconhecido"

#: module/plugins/hooks/ExtractArchive.py:317
msgid "Setting User and Group failed"
msgstr "Configuração de usuário e grupo falhou"

#: module/plugins/hooks/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'n'Load: porta 9666 já está em uso"

#: module/plugins/hooks/CaptchaTrader.py:70
#: module/plugins/hooks/Captcha9kw.py:60
#: module/plugins/hooks/ExpertDecoders.py:51
#, python-format
msgid "%s credits left"
msgstr "%s créditos sobrando"

#: module/plugins/hooks/CaptchaTrader.py:118
msgid "Could not send response."
msgstr "Não pode enviar resposta."

#: module/plugins/hooks/CaptchaTrader.py:136
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Sua conta CaptchaTrader não tem créditos suficientes"

#: module/plugins/hooks/LinkdecrypterCom.py:45
msgid "Crypter list not found"
msgstr "Lista do Crypter não encontrada"

#: module/plugins/hooks/LinkdecrypterCom.py:59
msgid "Crypter list is empty"
msgstr "Lista do Crypter vazia"

#: module/plugins/hooks/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Download terminado: %(name)s @ %(plugin)s"

#: module/plugins/hooks/Captcha9kw.py:94
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Novo CaptchaID do upload: %s : %s"

#: module/plugins/hooks/Captcha9kw.py:130
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Sua conta 9kw.eu não tem créditos suficientes"

#: module/plugins/hooks/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "scripts instalados para %s: "

#: module/plugins/hooks/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Scrip não executável:"

#: module/plugins/hooks/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Erro em %(script)s: %(error)s"

#: module/plugins/hooks/ExpertDecoders.py:96
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Sua conta ExpertDecoders não tem créditos suficientes"

#: module/plugins/hooks/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Por favor adicione sua conta rehost.to antes de reiniciar o pyLoad"

#: module/plugins/hooks/PremiumizeMe.py:50
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Por favor, adicione uma conta válida premiumize.me primeiro e reinicie o pyLoad."

#: module/plugins/hooks/CaptchaBrotherhood.py:70
#, python-format
msgid "%d credits left"
msgstr "%d créditos sobrando"

#: module/plugins/Plugin.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil e tesseract não estão instalados e nenhum cliente está conectado para reconhecimento de captcha"

#: module/plugins/Plugin.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Nenhum resultado de captcha obtido no tempo adequado, por todos os plugins."

#: module/plugins/Plugin.py:498 module/plugins/Plugin.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Configuração de usuário e grupo falhou: %s"

#: module/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Nenhum cliente conectado para descriptografar o captcha"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Adicionado pacote %(name)s contendo %(count)d links"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "Adicionado %(count)d links de pacote #%(package)d "

#: module/common/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Nenhuma engine js foi detectada, por favor instale o Spidermonkey, ossp-js, pyv8, nodejs ou rhino"

